src/components/pages/devices/flyouts/deviceDetails/deviceDetails.js

```diff
@@ -174,6 +175,15 @@ export class DeviceDetails extends Component {
     const tags = Object.entries(device.tags || {});
     const properties = Object.entries(device.properties || {});
 
+    // Add parameters to Time Series Insights Url
+    const timeSeriesParamUrl =
+    timeSeriesExplorerUrl
+      ? timeSeriesExplorerUrl +
```

Comment:
Consider more readable syntax (I'm sure there's a name for it, but I don't know what it is....) instead of concatenation.
Example: instead of 
```
'/maintenance/job/' + this.state.jobId
```
do this...
```
`/maintenance/job/${this.state.jobId}`
```